CHANGELOG.md

```diff
@@ -1,5 +1,64 @@
 # CHANGELOG
 
+## 2.0.0 (2022-11-14)
+
+MLflow 2.0.0 includes several breaking changes from the 1.x API, additional major features and improvements
+
+Breaking Changes:
+
+- [Recipes] Renamed `mlflow.pipelines` to `mlflow.recipes`
+- [Tracking] Remove deprecated `list` APIs for experiments, models, and runs from Python, Java, JS, and R APIs (#6785, #6786, #6787, #6788, #6800, #6868, @dbczumar)
+- [Tracking] Remove deprecated `runs` data from `get_experiment` REST API response (#6541, #6524 @dbczumar)
+- [Tracking] Remove deprecated `MLflowClient.download_artifacts` API (#6537, @WeichenXu123)
+- [Tracking] Add support for experiment creation resolution and validation for `MLFLOW_EXPERIMENT_NAME` and `MLFLOW_EXPERIMENT_ID` environment variables (#6674, @BenWilson2)
```

Comment:
Can we simplify this and make it a bit clearer what we fixed? I.e. `MLFLOW_EXPERIMENT_NAME` is now always respected even if the experiment did not previously exist